app/views/npq_separation/admin/applications/index.html.erb

```diff
@@ -1,3 +1,24 @@
+<% content_for :side_navigation do %>
+  <%=
+    render NpqSeparation::SubNavigationComponent.new(
+      request.path,
+      structure:
+        [
+          NpqSeparation::NavigationStructure::Node.new(
+            name: "All applications",
+            href: npq_separation_admin_applications_path,
+            prefix: /\/npq-separation\/admin\/applications(?!\/reviews)$/,
+            ),
+          NpqSeparation::NavigationStructure::Node.new(
+            name: "In review",
+            href: npq_separation_admin_application_reviews_path,
+            prefix: /\/npq-separation\/admin\/applications\/reviews$/,
+            ),
+        ]
```

Comment:
Can this be moved to a partial and be reused for the screen with the same code below, thanks